config/config.sample.yaml

```diff
@@ -4,15 +4,15 @@ listen: "0.0.0.0:8080"
 base_url: "https://op-tools-address.com"
 plugins:
   - name: bash_useful
-    file: plugins/bash.so
+    file: dist/plugins/bash.so
```

Comment:
Sample configuration is changed since build generates plugins in dist folder at repository. So `make go-run` will work when user executes the project from code base.